promptsource/templates/lince/sa_spaeng/templates.yaml

```diff
@@ -0,0 +1,100 @@
+dataset: lince
+subset: sa_spaeng
+templates:
+  24eba864-8859-4a15-a7a9-0fdf42d9f6cf: !Template
+    answer_choices: positive ||| negative ||| neutral
+    id: 24eba864-8859-4a15-a7a9-0fdf42d9f6cf
+    jinja: '{{words | join(" ") }}. What is the sentiment expressed by the original
+      poster? ||| {{ sa }}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - Accuracy
+      original_task: true
+    name: original poster expressed sentiment
+    reference: imdb
+  29756194-c221-48b8-8d25-db1e681d9eeb: !Template
+    answer_choices: positive ||| negative ||| neutral
+    id: 29756194-c221-48b8-8d25-db1e681d9eeb
+    jinja: '{{words | join(" ") }}. What sentiment is this post trying to express?
+      ||| {{ sa }}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - Accuracy
+      original_task: true
+    name: sentiment trying to express
+    reference: ''
+  52708ad1-0029-4d97-a5e9-e179da16e452: !Template
+    answer_choices: positive ||| negative ||| neutral
+    id: 52708ad1-0029-4d97-a5e9-e179da16e452
+    jinja: "{{ words | join(\" \") }}. This is definitely \n||| \n{% if sa == \"negative\"\
+      \ %} \nnot a positive post. \n{% elif sa == \"positive\" %} \nnot a negative\
+      \ post. \n{% else %} \na neutral post.\n{% endif %}"
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - Accuracy
+      original_task: false
+    name: negation template
+    reference: imdb
```

Comment:
Hello, @RosenZhang ! The `answer_choices` for `negation template` do not align with the designed targets. E.g. the `positive` in `answer_choices` should actually be `not a negative`. Is there any chance we could get a fix for this? Thanks!

Reply:
Thanks for the catch, just open a new PR to fix this! Let me know if that works!